issue_release_status.go

```diff
@@ -173,6 +174,17 @@ func retrieveFromReleaseManager(endpoint string, authToken string, output interf
 	req.Header.Add("Authorization", "Bearer "+authToken)
 
 	resp, err := httpClient.Do(req)
+	// Retry 4 times if error
+	if err != nil {
+		waitTimesSec := [4]int{10, 30, 120, 240}
+		for i := 0; i < 4; i++ {
+			time.Sleep(time.Duration(waitTimesSec[i]) * time.Second) // Sleep waitTimesSec seconds
```

Comment:
It is rather rare to see a Go `array` :) We usually only use slices for the benefit of not having to align the counter and size.

```suggestion
		waitTimesSec := []int{10, 30, 120, 240}
		for _, seconds := range waitTimesSec {
			time.Sleep(time.Duration(seconds) * time.Second) // Sleep waitTimesSec seconds
```

Further more I think these sleeps are pretty long? Waiting in total 6.6 minutes (400 seconds) before stopping makes little sens in our setup. I think GitHubs webhook timeout is 60 seconds. :) 

Maybe decrease to `1, 2, 5, 10` instead?

Another concern, that is not applicable here but is in other systems, is [the thundering herd problem](https://en.wikipedia.org/wiki/Thundering_herd_problem). If we get 10 webhooks and the release-manager is down, they all wait the same amount before trying again, leading to 4 waves of requests all leading to the same load on the service. To compensate for that, one usually add some jitter to distribute the load, say wait x seconds with a jitter of up to 1 second less or more which is randomly selected. :)